Sources/SwiftLexicalLookup/SwiftLexicalLookup.docc/SwiftLexicalLookup.md

```diff
@@ -0,0 +1,44 @@
+# SwiftLexicalLookup
+
+A library for performing lexical queries on Swift syntax trees.
+
+## Overview
+
+> Important:
```

Comment:
Please double-check that this works but I think docc supports `[!IMPORTANT]`, which would add some color here

```suggestion
> [!IMPORTANT]
```

Reply:
Sadly it doesn't seem to work. It just creates a plain note and embeds `[!IMPORTANT]` in the rest of the text. 
<img width="1000" alt="Zrzut ekranu 2024-08-21 o 19 00 51" src="https://github.com/user-attachments/assets/127e0501-4671-41d1-a75c-d79438168e5b">
I've also tried `[!IMPORTANT]:`, `**Important:**` and `**Important**:`, but haven't had much luck with them either.

Reply:
Oh, apparently the format is

```md
> Important: Your text
```

Like here: https://github.com/swiftlang/swift-docc/blob/main/Sources/docc/DocCDocumentation.docc/adding-structure-to-your-documentation-pages.md?plain=1#L51

Rendered as seen here: https://www.swift.org/documentation/docc/adding-structure-to-your-documentation-pages

Reply:
Ok, I made it inline.